src/NuGet.Clients/NuGet.SolutionRestoreManager/VSNominationUtilities.cs

```diff
@@ -246,6 +258,12 @@ private static bool IsLockFileFreezeOnRestore(IEnumerable tfms)
             return GetSingleNonEvaluatedPropertyOrNull(tfms, ProjectBuildProperties.RestoreLockedMode, MSBuildStringUtility.IsTrue);
         }
 
+        internal static bool IsCentralPackageVersionManagementEnabled(IEnumerable tfms)
+        {
+            // If the property is not defined the default value will be disabled. 
```

Comment:
Would you mind converting this comment to a Method Comment Block (///) which explains that <return> will be disabled by default?

Reply:
Updated